packages/uikit-workshop/src/scripts/components/pl-nav/nav-dropdown.scss

```diff
@@ -0,0 +1,42 @@
+@import '../../../sass/scss/core.scss';
+
+/**
+ * Nav Dropdown
+ * 1) On larger screens, display as dropdowns that
+ *    hangs over the header
+ */
+ .pl-c-nav__dropdown {
+  @media all and (min-width: $pl-bp-med) {
+    position: absolute;
+    top: 100%; /* 1 */
+    left: 0;
+    min-width: 12rem;
+    border-radius: $pl-border-radius-med;
+    border-style: solid;
+    border-width: 1px;
+    box-shadow: 0 2px 5px rgba(0,0,0,.1);
+    transition: all .2s ease;
+    transform: translateY(-12px);
+    z-index: 1;
+    transition: all $pl-animate-quick ease-out;
+
+    .pl-c-body--theme-sidebar & {
+      position: relative;
+    }
+  }
+
+  .pl-c-body--theme-light & {
+    border-color: rgba($pl-color-black, .2);
```

Comment:
Noticed that in [nav-link.scss](https://github.com/pattern-lab/patternlab-node/pull/1102/commits/4af8abfbed681ac4edff8f88aef99f2b5d90f29b#diff-6581b36fcb1402f0c8df623933ca75a2R20), a leading `0` is included on decimals, but they do not include the leading `0` here. 

I recommend these be consistent.